src/wallet/hardwaresigner/mod.rs

```diff
@@ -0,0 +1,17 @@
+// Bitcoin Dev Kit
+// Written in 2020 by Alekos Filini <alekos.filini@gmail.com>
+//
+// Copyright (c) 2020-2021 Bitcoin Dev Kit Developers
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Hardware wallets
+//!
+//! This module contains modules required for working with Hardware Wallets.
+//! This module can contain various vendor specific code, if required.
+
+pub mod signer;
```

Comment:
Why we need a separate module folder for this? Are we expecting more signer modules in future?

Reply:
Yes, I think there should be more signers/functions, specifically for hardware wallets, in the future. Hardware wallets differ greatly in function (even vis-a-vis HWI) and to take full advantage of them, we would need some vendor/device specific code in both rust-hwi and BDK in the future.

Reply:
That makes sense.. Thanks..

Reply:
I'm not sure we'll ever add more signers, `HWI` is specifically built for handling the specific code for vendors/devices

Reply:
Yeah thats what I was thinking too.. IIUC HWI can handle any vendor's device if they support the API.. So in that case we should just need one module file to cover it..

Reply:
Okay. I will refactor the code into one file.